dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -342,8 +345,15 @@ private void writeFile(List<HPCCRecord> records, String fileName, FieldDef recor
                 }
 
                 HPCCRecordAccessor recordAccessor = new HPCCRecordAccessor(recordDef);
-                HPCCRemoteFileWriter<HPCCRecord> fileWriter = new HPCCRemoteFileWriter<HPCCRecord>(hpccPartitions[partitionIndex], recordDef,
-                        recordAccessor, CompressionAlgorithm.NONE);
+                HPCCRemoteFileWriter<HPCCRecord> fileWriter = null;
+                if (connectTimeoutMs != null) {
+                    fileWriter=new HPCCRemoteFileWriter<HPCCRecord>(hpccPartitions[partitionIndex], recordDef,
+                            recordAccessor, CompressionAlgorithm.NONE,connectTimeoutMs);
+                    Thread.sleep(2000);
```

Comment:
what are we accomplishing with this sleep?
Also, minor but formatting needed.

Reply:
the default timeout ms is 1000. If an override is specified, sleep longer than the default timeout to verify the overridden timeout is actually working.

Reply:
Fixed formatting